tools/acrn-crashlog/acrnprobe/README.rst

```diff
@@ -50,6 +50,19 @@ Syntax
   + polling: run a detecting job with fixed time interval.
   + inotify: monitor the change of file or dir.
 
+- trigger :
+  Essentially, trigger represents one section of content. It could be
+  a file's content, a directory's content, or a memory's content which can be
+  obtained. By monitoring it ``acrnprobe`` could detect the certain events which
```

Comment:
change ``the certain`` to just ``certain``

Reply:
Done.